data/json/items/bionics.json

```diff
@@ -733,6 +742,15 @@
     "price": 800000,
     "difficulty": 11
   },
+  {
+    "id": "bio_surgical_razor",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": "Autonomous Surgical Scalpels CBM",
+    "description": "Implanted on the user's fingers is a system of surgical grade scalpels.  They will allow you to make automated precise cuts, and are useful as a butchering tool.",
```

Comment:
Change the first sentence to "A system of surgical grade scalpels is implanted in the user's fingers."
Change the second sentence to "The user can make surgically precise cuts, which is also useful for butchery."